<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TransducerSelectorWidget</class>
 <widget class="QWidget" name="TransducerSelectorWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>131</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="labelLoadCell">
       <property name="text">
        <string>Load Cell:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxLoadCell"/>
     </item>
     <item>
      <widget class="QLabel" name="labelMaxForce">
       <property name="text">
        <string>Max: 0 N</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="labelPressureTransducer">
       <property name="text">
        <string>Pressure Transducer:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxPressureTransducer"/>
     </item>
     <item>
      <widget class="QLabel" name="labelMaxPressure">
       <property name="text">
        <string>Max: 0 Pa</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
